packages/frontend2/src/app/(new)/(other)/scaling/liveness/_components/table/scaling-liveness-table.tsx

```diff
@@ -0,0 +1,59 @@
+'use client'
+
+import { getCoreRowModel, getSortedRowModel } from '@tanstack/react-table'
+import { useMemo } from 'react'
+import { BaseScalingFilters } from '~/app/(new)/(other)/_components/base-scaling-filters'
+import { useScalingFilter } from '~/app/(new)/(other)/_components/scaling-filter-context'
+import { BasicTable } from '~/app/_components/table/basic-table'
+import { useTable } from '~/hooks/use-table'
+import { type ScalingLivenessEntry } from '~/server/features/scaling/get-scaling-liveness-entries'
+import { useLivenessTimeRangeContext } from '../liveness-time-range-context'
+import { LivenessTimeRangeControls } from '../liveness-time-range-controls'
+import { columns } from './columns'
+import { toLivenessTableEntry } from './to-table-entry'
+
+export interface Props {
+  entries: ScalingLivenessEntry[]
+}
+
+export function ScalingLivenessTable({ entries }: Props) {
+  const { timeRange, setTimeRange } = useLivenessTimeRangeContext()
+  const includeFilters = useScalingFilter()
+
+  const tableEntries = useMemo(
+    () =>
+      entries
+        .filter(includeFilters)
+        .map((item) => toLivenessTableEntry(item, timeRange)),
+    [entries, includeFilters, timeRange],
+  )
+
+  const table = useTable({
+    data: tableEntries,
+    columns: columns,
+    getCoreRowModel: getCoreRowModel(),
+    getSortedRowModel: getSortedRowModel(),
+    manualFiltering: true,
+    initialState: {
+      sorting: [
+        {
+          id: '#',
+          desc: false,
+        },
+      ],
+    },
+  })
+
+  return (
+    <section className="mt-4 space-y-6 sm:mt-8">
+      <div className="flex flex-col justify-between gap-4 md:flex-row">
+        <BaseScalingFilters items={entries} />
```

Comment:
Passing unfiltered entries to the filters causes the filter options to be too broad